test_type/shard.js

```diff
@@ -322,14 +326,32 @@ async function clear(testList) {
   console.log(`- Finish to cleanup data`);
 }
 
+async function checkAndWriteMonitoringInfo(benchmarkConfig, startTime, endTime, testResult) {
+  const monitoringConfig = benchmarkConfig.monitoring;
+  if (!monitoringConfig || !monitoringConfig.enable) {
+    return;
+  }
+  if (!monitoringConfig.projectId || !monitoringConfig.instanceName ||
+      !monitoringConfig.keyFilename) {
+    console.log(`Invalid monitoring config`);
+    return;
+  }
+
+  const monitoringInfo = await getMonitoringInfo(monitoringConfig.projectId,
+      monitoringConfig.instanceName, monitoringConfig.keyFilename, startTime, endTime);
+  testResult.monitoring = monitoringInfo;
```

Comment:
Minor: Can be merged with the above line?

Reply:
Changed due to other code fixes!